adapters/rhythmone/rhythmone.go

```diff
@@ -132,9 +132,9 @@ func (a *RhythmoneAdapter) preProcess(req *openrtb.BidRequest, errors []error) (
 			errors = append(errors, err)
 			return nil, "", errors
 		}
-		bidderExtCopy := openrtb_ext.ExtBid{
-			Bidder: rhythmoneExtCopy,
-		}
+		bidderExtCopy := struct {
+			Bidder json.RawMessage `json:"bidder,omitempty"`
+		}{rhythmoneExtCopy}
```

Comment:
This looks like a misuse of the `ExtBid` object, which is related to the response not the request. Replaced with inline anonymous struct.

Reply:
Nice